.github/workflows/on-pr-example.yml

```diff
@@ -0,0 +1,20 @@
+name: Run reusable workflow on pr's
+on: workflow_dispatch # replace with your trigger, probably pull_request
+
+# To write annotations and comments to the PR
+permissions: 
+  contents: read
+  checks: write
+  pull-requests: write
+
+concurrency: on-pr-example-concurrency
+
+jobs:
+  gql-lint:
+    uses: sketch-hq/gql-lint-integrations/.github/workflows/gql_new_deprecated_fields.yml@main
```

Comment:
This URL is old, right?

Reply:
🕵️  I was changing the others but forgot about this one :) Fix incoming

Reply:
https://github.com/sketch-hq/gql-lint/pull/34/commits/04069f5883d645a29b7dea1c4a68906b3af6c85f